libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -474,10 +482,28 @@ private void resetAnalyticsPolicy() {
         e.commit();
     }
 
-    private static void recreateAnalyticsPublishPeriodicWorkRequest() {
-        AnalyticsPublishingWorker.Companion.reEnqueueAnalyticsPublishPeriodicWorkRequest(
-                SalesforceSDKManager.getInstance().getAppContext(),
-                sPublishFrequencyInHours
-        );
+    private static ScheduledFuture<?> recreateAnalyticsPublishPeriodicWorkRequest() {
```

Comment:
Overall and since this is a temporary work-around, I accurately restored this to its 10.2 state.  I did add a little Java syntax cleanup and inspector warning resolution to the modified code.  I also added a detailed to-do as well to explain why this is here and document some suggestions for the future.